### PR TITLE
Emags will no longer work on Ratvar-serving cyborgs

### PR DESCRIPTION
**Pull request by ChangelingRain**
Also removes a tiny, difficult to notice tell from emagging a borg that would resist emagging.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
:-1:

**Comment by Jordie0608** [comment]
Whats the reason for this?

**Comment by XDTM** [comment]
Emagging a Ratvar borg will give it emagged laws, but it also keeps his Ratvar status, which means it can communicate and see the Ratvar HUD. This creates confusion over whether to follow laws or ratvar, and Ratvar's influence seemed stronger than that of a mortal-made circuit scrambler.

File: code/modules/mob/living/silicon/robot/robot_defense.dm [hunk 1]
```diff
@@ -80,75 +80,75 @@
...
-			sleep(10)
-			src << "<span class='danger'>> N</span>"
-			sleep(20)
-			src << "<span class='danger'>ERRORERRORERROR</span>"
-			src << "<b>Obey these laws:</b>"
-			laws.show_laws(src)
-			src << "<span class='danger'>ALERT: [user.real_name] is your new master. Obey your new laws and their commands.</span>"
-			SetLockdown(0)
-			update_icons()
+		return
```
Comment (ChemicalRascal) [inline code/modules/mob/living/silicon/robot/robot_defense.dm]: I'll admit to not being familiar with BYOND, but doesn't this imply that emag_act will only continue if the robot is emagging themselves? This, at least, is what the distinction between the original code and this code seems to show.

In other words: Are you sure you don't mean `user == src` here?

**Comment by ChemicalRascal** [comment]
Only because I care, Changeling, only because I care.

**Comment by Anonmare** [comment]
Shouldn't the Ratvar Law be the highest one in the sequence? If it is then there shouldn't be any confusion. 
Antag Laws > Regular Laws
Higher Laws > Lower Laws

File: code/modules/mob/living/silicon/robot/robot_defense.dm [hunk 2]
```diff
@@ -79,75 +79,76 @@
...
+			user << "<span class='warning'>The cover is already unlocked!</span>"
+		return
+	if((world.time - 100) < emag_cooldown)
+		return
+	if(wiresexposed)
+		user << "<span class='warning'>You must unexpose the wires first!</span>"
+		return
+
+	user << "<span class='notice'>You emag [src]'s interface.</span>"
+	emag_cooldown = world.time
```
Comment (optimumtact) [inline code/modules/mob/living/silicon/robot/robot_defense.dm]: can you change the check above to be world.time < emag_cooldown

and then change this line to emag_cooldown = world.time + 100

having them backwards is very confusing

**Pull request merged**